bootstrap-demo/kubernetes-api-java/build.sbt

```diff
@@ -0,0 +1,18 @@
+import com.typesafe.sbt.packager.docker._
```

Comment:
should we just remove this and have the scala one use sbt and the Java one use maven?